tests/sentry/event_manager/test_event_manager.py

```diff
@@ -2086,6 +2093,107 @@ def test_auto_update_grouping(self):
             project = Project.objects.get(id=self.project.id)
             assert project.get_option("sentry:grouping_config") == DEFAULT_GROUPING_CONFIG
 
+    @override_options({"store.use-ingest-performance-detection-only": 1.0})
+    @override_options({"performance.issues.all.problem-creation": 1.0})
+    @override_options({"performance.issues.all.problem-detection": 1.0})
+    def test_perf_issue_creation(self):
```

Comment:
I moved these tests here so that they'd mirror the location of eventmanager, sincethese all operate there.